internal/proxy_manager.go

```diff
@@ -98,6 +98,8 @@ func (proxyManager *ProxyManager) getProxyByNameSpace(serviceName *string, name
 		return newListProxy(proxyManager.client, serviceName, name)
 	} else if common.SERVICE_NAME_SET == *serviceName {
 		return newSetProxy(proxyManager.client, serviceName, name)
+	} else if common.SERVICE_NAME_MULTI_MAP == *serviceName {
+		return newMultiMapProxy(proxyManager.client, serviceName, name), nil
```

Comment:
is `nil` at the end optional? Because some proxies does not have it. Better to make it all the same.